### PR TITLE
feat: add cross on application to close modal

### PR DESCRIPTION
**Pull request by Narkfr**

### CODE REVIEW

**Comment by Narkfr** [comment]
[(3) [IA-1] Amélioration de la modale](https://trello.com/c/l9u0C1Ki/663-3-ia-1-am%C3%A9lioration-de-la-modale)

File: src/views/steps/subviews/service.coffee [hunk 1]
```diff
@@ -55,7 +55,7 @@ module.exports = class ServiceView extends ItemView
 
 
         cozy.client.intents
-            .create 'CREATE', 'io.cozy.accounts', slug: @slug
+            .create 'CREATE', 'io.cozy.accounts', { slug: @slug, withCross: false }
```
Comment (CPatchane) [inline src/views/steps/subviews/service.coffee:58]: We're still using `withCross`? 😨

Comment (ptbrowne) [reply src/views/steps/subviews/service.coffee:58]: Yes, this issue is still pending https://github.com/cozy/cozy-ui/issues/198

Comment (Narkfr) [reply src/views/steps/subviews/service.coffee:58]: That's not the same `withCross`, this one is for the service, not for the modal :) and it's temporary

Comment (ptbrowne) [reply src/views/steps/subviews/service.coffee:58]: https://github.com/cozy/cozy-ui/pull/237 ;)

**Pull request merged**